application/models/user_access_token.go

```diff
@@ -102,3 +106,39 @@ func (u *UserAccessToken) GetOrganization() (Organization, error) {
 
 	return uOrg.Organization, nil
 }
+
+func createAccessTokenExpiry() time.Time {
+	lifetime := envy.Get("ACCESS_TOKEN_LIFETIME", "3600")
```

Comment:
Want to make the default for this and `lifetimeSeconds` constants that match?

Reply:
Good call. Added a const to domain.go.